tools/Azure.Mcp.Tools.SignalR/src/Commands/Identity/IdentityListCommand.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Mcp.Core.Commands;
+using Azure.Mcp.Tools.SignalR.Options;
+using Azure.Mcp.Tools.SignalR.Options.Identity;
+using Azure.Mcp.Tools.SignalR.Services;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Mcp.Tools.SignalR.Commands.Identity;
+
+/// <summary>
+/// List the managed identity configuration of an Azure SignalR Service.
+/// </summary>
+public sealed class IdentityListCommand(ILogger<IdentityListCommand> logger)
+    : BaseSignalRCommand<IdentityListOptions>
+{
+    private const string CommandTitle = "List Identity Configuration";
+    private readonly ILogger<IdentityListCommand> _logger = logger;
+
+    private static readonly Option<string> _signalRNameOption = SignalROptionDefinitions.SignalR;
+
+    public override string Name => "list";
+
+    public override string Description =>
+        """
+        List the managed identity configuration of an Azure SignalR Service. Returns identity information
+        including type (SystemAssigned, UserAssigned), principal ID, tenant ID, and any
+        user-assigned identities associated with the service.
+        """;
+
+    public override string Title => CommandTitle;
+
+    public override ToolMetadata Metadata => new() { Destructive = false, ReadOnly = true, Secret = true };
```

Comment:
Please provide values for all hints.

https://github.com/microsoft/mcp/blob/main/docs/new-command.md#destructive-property